Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1

```diff
@@ -467,61 +560,96 @@ function Remove-AnonymousAccessOnContainers
-                            Write-Host "Use this file for any roll back that may be required." -ForegroundColor $([Constants]::MessageType.Info)
-                        }
+                            if ($($remediatedStorageAccounts | Measure-Object).Count -gt 0)
+                            {
+                                Write-Host $([Constants]::SingleDashLine)
+                                # Write this to a file.
+                                $storageAccountsRemediatedFile = "$($folderpath)\RemediatedStorageAccounts.csv"
+                                $remediatedStorageAccounts | Export-CSV -Path $storageAccountsRemediatedFile -NoTypeInformation
+                                Write-Host "The information of Storage accounts where 'Allow Blob Public Access' has been successfully disabled is saved to $($storageAccountsRemediatedFile)"
+                                Write-Host "Use this file for any roll back that may be required." -ForegroundColor $([Constants]::MessageType.Info)
+                                
+                            }
 
-                        if ($($skippedStorageAccountsFromRemediation | Measure-Object).Count -gt 0)
-                        {
-                            Write-Host $([Constants]::SingleDashLine)
-                            Write-Host "`nError disabling 'Allow Blob Public Access' for the following Storage account(s):" -ForegroundColor $([Constants]::MessageType.Error)
-                            $skippedStorageAccountsFromRemediation | Select-Object -Property ResourceGroupName , StorageAccountName , ResourceId | ft
-            
-                            # Write this to a file.
-                            $storageAccountsSkippedFile = "$($folderpath)\SkippedStorageAccountsFromRemediation.csv"
-                            $skippedStorageAccountsFromRemediation | Export-CSV -Path $storageAccountsSkippedFile -NoTypeInformation
-                            Write-Host "This information has been saved to $($storageAccountsSkippedFile)"
+                            if ($($skippedStorageAccountsFromRemediation | Measure-Object).Count -gt 0)
+                            {
+                                Write-Host $([Constants]::SingleDashLine)
+                                # Write this to a file.
+                                $storageAccountsSkippedFile = "$($folderpath)\SkippedStorageAccountsFromRemediation.csv"
+                                $skippedStorageAccountsFromRemediation | Export-CSV -Path $storageAccountsSkippedFile -NoTypeInformation
+                                Write-Host "The information of Storage accounts where 'Allow Blob Public Access' has not been successfully disabled is saved to $($storageAccountsSkippedFile)"
+                                
+                            }
                         }
-                        
-                        return 
```

Comment:
Why this return statement removed this was in normal flow path?

Reply:
It was not necessary, so removed it.